backend/lib/edgehog/containers/deployment/changes/check_containers.ex

```diff
@@ -22,22 +22,17 @@ defmodule Edgehog.Containers.Deployment.Changes.CheckContainers do
   @moduledoc false
   use Ash.Resource.Change
 
-  alias Edgehog.Devices
-
   @impl Ash.Resource.Change
-  def change(changeset, _opts, context) do
-    %{tenant: tenant} = context
+  def change(changeset, _opts, _context) do
     deployment = changeset.data
 
     with {:ok, :created_networks} <- Ash.Changeset.fetch_argument_or_change(changeset, :status),
          {:ok, deployment} <-
-           Ash.load(deployment, [:device, release: [:containers]], reuse_values?: true),
-         {:ok, available_containers} <-
-           Devices.available_containers(deployment.device, tenant: tenant) do
-      available_containers = Enum.map(available_containers, & &1.id)
+           Ash.load(deployment, [device: :available_containers, release: [:containers]], reuse_values?: true) do
```

Comment:
this will not work as expected: if we use the same `load` call with `reuse_values?: true`, we won't load the updated resources from the device

```suggestion
           Ash.load(deployment, [device: :available_containers, release: [:containers]]) do
```